docs/configuration/network_filters/mongo_proxy_filter.rst

```diff
@@ -25,6 +26,30 @@ access_log
   path is specified no access logs will be written. Note that access log is also gated by
   :ref:`runtime <config_network_filters_mongo_proxy_runtime>`.
 
+fault
+  *(optional, object)* If specified, the filter will inject faults based on the values in the object.
+
+Fault configuration
+-------------------  
+
+Configuration for fixed duration delays into `Query` mongo operation. 
```

Comment:
About this configuration, what will the other delays ones look like? If we are to allow per operation configuration, should the name of query be in here.